docs/DeveloperGuide.md

```diff
@@ -132,6 +132,30 @@ The `viewImageWindow` is the displayed after it retrieves the set from `model`
 
 Step 5: The set of `UserImage` is then converted into a `ArrayList` and the first image is displayed in the window.
 
+## [Proposed] Sorting
+
+The sorting feature allows the user to sort the list of `Person` displayed.
+
+The following table shows the attributes that the list can be sorted by and their corresponding keywords.
+
+| Attribute            | Keyword        |
+|----------------------|----------------|
+| `Name`               | `name`         |
+| `Phone`              | `phone`        |
+| `Email`              | `email`        |
+| `Favourite`          | `favourite`    |
+| `Address`            | `address`      |
+| `UserType`           | `usertype`     |
+| Number of `Property` | `num_property` |
+
+Sorting the list is done by using the `sort` command, which has the following syntax: `sort [KEYWORD]...`.
+
+If multiple attributes are specified, the first attribute is given the highest priority, while the last attribute is given the lowest priority. For example, `sort address name` will sort the list by `Address` first, followed by `Name` if `Address` is equal.
```

Comment:
Maybe use another field rather than address, since we made it clear that every buyer/seller can have 1 address right?

Reply:
What this means is that if 2 people have the same `address`, they will be sorted according to `name` instead.